src/user/entities/user.entity.ts

```diff
@@ -1,16 +1,23 @@
+import { Field, ObjectType } from '@nestjs/graphql';
 import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
 
+@ObjectType()
 @Entity()
 export class User {
+
+  @Field()
   @PrimaryGeneratedColumn()
```

Comment:
@NanditaNikam do not use `@ObjectType` in entity...make separate output type for query and mutations

Reply:
ok